src/main/java/com/thebois/views/StructureView.java

```diff
@@ -0,0 +1,56 @@
+package com.thebois.views;
+
+import java.util.ArrayList;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+
+import com.thebois.models.world.structures.IStructure;
+
+/**
+ * Responsible for displaying structures.
+ */
+public final class StructureView implements IView {
+
+    private final float tileSize;
+    private ArrayList<IStructure> structures = new ArrayList<>();
+    private final Color brown = new Color(0.4f, 0.2f, 0f, 1);
+
+    /**
+     * Creates an instance of a Structure view.
+     *
+     * @param tileSize The tile size of the world in order to render structures in the right size.
+     */
+    public StructureView(final float tileSize) {
+        this.tileSize = tileSize;
+    }
+
+    @Override
+    public void draw(ShapeRenderer batch) {
+        for (IStructure structure : structures) {
+            batch.setColor(brown);
+            final int posX = (int) ((structure.getPosition().getPosX()) * tileSize);
+            final int poxY = (int) ((structure.getPosition().getPosY()) * tileSize);
+            batch.rect(posX, poxY, tileSize, tileSize);
+        }
+    }
+
+    /**
+     * Gets the tile size of the world.
+     *
+     * @return the tile size as a float.
+     */
+    public float getTileSize() {
+        return tileSize;
+    }
+
+    /**
+     * Updates the structures that should be displayed in the game.
+     *
+     * @param Structures The structures that are to be displayed.
+     */
+    public void updateStructureView(ArrayList<IStructure> Structures) {
```

Comment:
The class is called `StructureView` so calling `StructureView.updateStructureView` is quite redundant. Better to just call it `update` since we will be calling `StructureView.update`.

Reply:
Oh, and the argument shouldn't be capitalized, it should be `structures`.

Reply:
Changing it.

Reply:
> Oh, and the argument shouldn't be capitalized, it should be `structures`.

@martinjonsson01 
![image](https://user-images.githubusercontent.com/55716109/134013368-6cf9ac4d-d539-4948-a715-a97fa0e78639.png)
I still feel like structures is a good argument name, opinion on this checkstyle error?

Reply:
Hmm, normally it's good that it warns about hiding fields as it's simple to miss and can cause confusing issues. In this case we have a method that is acting like a setter, but isn't named like one, so checkstyle doesn't allow it the exemption it does for setters.

I'd call it something like `newStructures` or `updatedStructures`. We did something similar in the update method for `ITerrain`.